cmd/evm/internal/t8ntool/transition.go

```diff
@@ -252,7 +252,7 @@ func Transition(ctx *cli.Context) error {
 			// Already set, base fee has precedent over parent base fee.
 		} else if prestate.Env.ParentBaseFee != nil {
 			parent := &types.Header{
-				Number:   new(big.Int).SetUint64(prestate.Env.Number),
+				Number:   new(big.Int).SetUint64(prestate.Env.Number - 1),
```

Comment:
If we don't already, we need a pre-check that this isn't `0`, otherwise the parent would wind up at `0xffff..f`.

Reply:
I just figured that the `ParentBaseFee` in the input, makes little sense if the block number is `0`, but on second thought, it is better to explicitly perform this check. Have updated it.